packages/kbn-optimizer/README.md

```diff
@@ -0,0 +1,102 @@
+# @kbn/optimizer
+
+`@kbn/optimizer` is a package for building new platform UI plugins (and hopefully more soon).
+
+New Platform plugins with `"ui": true` in their `kibana.json` file will have their `public/index.ts` file (and all of its dependencies) bundled into the `target/public` directory of the plugin. The build output does not need to be updated when other plugins are updated and is included in the distributable without requiring that we ship `@kbn/optimizer` 🎉.
```

Comment:
nit: I believe Platform team is renaming "New Platform" to "Kibana Platform".

Reply:
I'm not opposed to something like this, but there are a lot of places where I'm referring to "new platform plugins" in this code, like the script to build them for instance. @joshdover do you have an opinion about what I should update to the new "Kibana Platform Plugins" verbiage?